plugins/src/test/java/com/qubole/quark/plugins/jdbc/JdbcFactoryValidationTest.java

```diff
@@ -33,7 +33,7 @@ public void testGenericDb() throws QuarkException {
   public void testValidProps() throws QuarkException {
     Map<String, Object> props = new HashMap<>();
     props.put("type", "mysql");
-    props.put("url", "jdbc:quark:test");
+    props.put("url", "fatjdbc:quark:test");
```

Comment:
Have you changed the connect string for fat jar from jdbc:quark to fatjdbc:quark ?